index.html

```diff
@@ -162,12 +162,37 @@ <h1 class="chunk alignCenter">micropolisJS</h1>
           </div>
         </noscript>
       </main>
-      <footer id="footer" class="alignCenter chunk white padding10">
```

Comment:
Same with this. Maybe you want to keep this out of pull requests and leave it in your personal clone? You can archive this by maintaining different branches. For example `develop` is your local code and for a PR you create a new branch based on "upstream" (this) repository.

Exexecute: `git remote add upstream https://github.com/graememcc/micropolisJS.git` to add it.